eng/Version.Details.xml

```diff
@@ -29,14 +29,18 @@
       <Uri>https://github.com/dotnet/symreader</Uri>
       <Sha>27e584661980ee6d82c419a2a471ae505b7d122e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging" Version="8.0.0-rtm.23517.16">
+    <Dependency Name="Microsoft.Extensions.Logging" Version="8.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>6f7af556d2761b0c93299cb88c61e4b747d6176a</Sha>
-      <SourceBuild RepoName="runtime" />
+      <Sha>488a8a3521610422e8fbe22d5cc66127f3dce3dc</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.FileSystemGlobbing" Version="8.0.0">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>488a8a3521610422e8fbe22d5cc66127f3dce3dc</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.FileSystemGlobbing" Version="8.0.0-rtm.23517.16">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.runtime.linux-x64" Version="8.0.0-rtm.23523.12">
```

Comment:
I think the right thing to do here is to just remove the `SourceBuild` tag for runtime as was done and then you shouldn't need this `Microsoft.SourceBuild.Intermediate.runtime.linux-x64` dependency at all.

Reply:
We don't need the sourcebuild intermediate package? Not even in main?

Reply:
The dependencies have been changed to a stable version so it shouldn't require it.